release_tester/arangodb/installers/deb.py

```diff
@@ -93,7 +93,11 @@ def upgrade_package(self):
         server_upgrade = pexpect.spawnu('dpkg -i ' +
                                         str(self.cfg.package_dir / self.server_package))
         try:
-            server_upgrade.expect('Upgrading database files')
```

Comment:
please harmonize the log statements as discussed. Otherwise LGTM.

Reply:
Harmonized the log statements.